docs/api-reference/deck.md

```diff
@@ -179,16 +179,20 @@ background-color: #29323c;
 padding: 10px;
 ```
 
-##### `useDevicePixels` (Boolean, optional)
+##### `useDevicePixels` (Boolean or Number, optional)
+
+When `true` (default), it will use device's full retina/HD resolution for rendering, when `false`, it will use the same resolution as the screen window (CSS window).
```

Comment:
Use a list:
- `true`:
- `false`:
- `<number>`:

Instead of `device's full retina/HD resolution`, just link to https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio

Is `CSS window` a thing? Don't we just use `1` when `useDevicePixels: false`?

Reply:
I will updated to `... when false, it will use the CSS pixels resolution`.

Reply:
`CSS pixels resolution` doesn't really mean anything. I recommend sticking with the term "device pixel ratio" which people can find plenty of explanations online.

Reply:
We need to talk about two resolutions, `CSS pixels` resolution and `Device (physical) pixels` resolution as referred in MDN docs, I updated the PR accordingly.